### PR TITLE
chore: update README.md default options & copyright year

### PR DESCRIPTION
**Pull request by yuler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ $ npm install arg
 `arg()` takes either 1 or 2 arguments:
 
 1. Command line specification object (see below)
-2. Parse options (_Optional_, defaults to `{permissive: false, argv: process.argv.slice(2)}`)
+2. Parse options (_Optional_, defaults to `{permissive: false, permissive: false, argv: process.argv.slice(2)}`)
```
Comment (Qix-) [inline README.md]: `permissive: false` is already there...

Comment (yuler) [reply README.md]: sorry typo is `stopAtPositional`

Comment (Qix-) [reply README.md]: Aha, I see :)

**Comment by Qix-** [comment]
Thanks ^^

**Comment by Qix-** [comment]
Released as `4.1.2`. :)

**Pull request merged**